ca/certificate-authority_test.go

```diff
@@ -436,6 +437,8 @@ func TestRevoke(t *testing.T) {
 	if err != nil {
 		return
 	}
+	pub, _ := publisher.NewPublisherAuthorityImpl(nil)
```

Comment:
Rather than use a real PublisherAuthority here, please use a mock. It doesn't have to go in mocks/, you can define it in this file:

```
type MockPublisherAuthority struct{};
func (*MockPublisherAuthority) SubmitToCT([]byte) error {
  return nil
}
```